src/main/java/org/fundacion/jala/converter/models/Asset.java

```diff
@@ -55,88 +56,99 @@ public Asset(final String newTitle, final String newPath, final String newChecks
     }
 
     /**
-     * Obtains Id of the project
-     * @return id of the project
+     * Obtains Id of the database.
+     *
+     * @return id of the audio or video converter.
      */
     public int getId() {
         return id;
     }
 
     /**
-     * Sets new id of the project
-     * @param newId int with the new project id
+     * Sets new id of the audio or video converter.
+     *
+     * @param newId is a int if the new audio or video converter.
      */
     public void setId(final int newId) {
         this.id = newId;
     }
 
     /**
-     * Obtains title of the project
-     * @return title of the project
+     * Obtains title of the audio or video converter.
+     *
+     * @return title of the project.
      */
     public String getTitle() {
         return title;
     }
 
     /**
-     * Sets new title of the project
-     * @param newTitle String with the new title
+     * Sets new title of the audio or video converter.
+     *
+     * @param newTitle String with the new title.
      */
     public void setTitle(final String newTitle) {
         this.title = newTitle;
     }
 
     /**
-     * Obtains the user that realized the project
-     * @return user
+     * Obtains the user that realized the project.
+     *
+     * @return name of the user that is used the audio or video converter.
```

Comment:
you should remove the is from the sentence.

Reply:
ok

it was solved